src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java

```diff
@@ -116,8 +116,8 @@ public void service(final HttpServletRequest req, final HttpServletResponse resp
         log.info("Scheduling Hystrix command with key '" + key + "' for path: '" + path + "'.");
 
         //double thread pool size for pool with magic name 'default'
-        final int size = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
-        final int queueSize = (int) (((double) corePoolSize) * 1.4d);
+        final int coreSize = DEFAULT_COMMAND_GROUP_KEY.equals(key) ? (corePoolSize * 2) : corePoolSize;
```

Comment:
Might as well revert this properly (rename the variable again, and fix whitespace on line 120)

Reply:
Personally I think this alignment makes for better readability. No accounting for taste though, right? :)

Reply:
IMHO less noise in pull requests is always preferred. We want to see what you wanted to change, not what just changed due to personal taste or accidents.

Reply:
IMO there's a difference between indenting a single line and reformatting in a way that obfuscates functional changes. We're contributing more (still negligible) noise just by having this discussion :P

Reply:
No, we're not. "git blame" will return the wrong thing by merging this change. Long-term, you would want "git blame" to return something useful.

Reply:
Fair enough. I agree with everything you say, but I think you're making it out to be a bigger problem than it really is. Nitpicking over miniscule issues is generally not all that constructive (perfect is the enemy of good etc.). Feel free to disagree.